src/cmd/manage_nsfs.js

```diff
@@ -414,8 +415,14 @@ async function fetch_account_data(argv, from_file) {
     // secret_key as SensitiveString
     data.access_keys[0].secret_key = _.isUndefined(data.access_keys[0].secret_key) ? undefined :
         new SensitiveString(String(data.access_keys[0].secret_key));
-    // allow_bucket_creation infer from new_buckets_path
-    data.allow_bucket_creation = !_.isUndefined(data.nsfs_account_config.new_buckets_path);
+    // allow_bucket_creation either set by user or infer from new_buckets_path
+    if (_.isUndefined(argv.allow_bucket_creation)) {
+        data.allow_bucket_creation = !_.isUndefined(data.nsfs_account_config.new_buckets_path);
+    } else if (typeof argv.allow_bucket_creation === 'string') {
```

Comment:
```suggestion
    } else if (ALLOW_BUCKET_CREATION_VALUES.includes(argv.allow_bucket_creation.toLowerCase()) {
      data.allow_bucket_creation = argv.allow_bucket_creation.toLowerCase();
      }
```

Reply:
I suggested to explicitly check it so the reader will understand the values and not just the type

Reply:
We are checking in in the function `validate_options_type_by_value` - type and value.
The above suggested change is in the `fetch_account_data`, which is called later.